operations/research-and-development/product/how-to-guides-for-product/release-announcement.md

```diff
@@ -30,15 +30,15 @@ A release announcement for each feature should be started early in the product p
 
 #### D. \(T-minus 7 working days\) Product Marketing Director Review
 
-* The PM-approved draft will be provided for review to the Director of Product Marketing. Katie or Jason provides the release announcement copy for review and relays edits back to the Product Management team via the Product Manager's internal channel.
+* The PM-approved draft will be provided for review to the Director of Product Marketing. Katie provides the release announcement copy for review and relays edits back to the Product Management team via the Product Manager's internal channel.
```

Comment:
this is internal documentation? make we can give full names to avoid confusion. the ideal is not give name at all, but if is needed I would like to see the full name (and the email or the username for the community server)

Reply:
Added full name + username)